modules/hydra-master.nix

```diff
@@ -26,6 +27,8 @@ in {
     mode = "0440";
   };
 
+  environment.systemPackages = [ iohk-pkgs.iohk-ops ];
```

Comment:
It absolutely doesn't -- but previously you have added `iohk-ops` to _all_ machines.

I just made it so that it's not deployed on the cluster -- retaining other places, as per your original design.

Tell me if I should drop it from Hydra as well.

Reply:
Dropped.

Reply:
The question was more complicated, apparently, as we've reversed our decision here once more.